libs/newsletter/src/driver/hubspot/newsletter.service.spec.ts

```diff
@@ -59,16 +69,71 @@ describe('DaffHubspotNewsletterService', () => {
 
       req.flush('');
     })));
-    it('should call the Title service', async(inject([Title], (title) => {
-      spyOn(title, 'getTitle').and.callThrough();
+    it('should transform the response recieved from Hubspot', async(inject([DaffNewsletterTransformer], (transformer) => {
+      spyOn(transformer, 'transformIn').and.callThrough();
       const submission = { email: 'email@email.com', firstName: 'John', lastName: 'Doe' };
       newsletterService.send(submission).subscribe(() => {
-        expect(title.getTitle).toHaveBeenCalled();
+        expect(transformer.transformIn).toHaveBeenCalled();
       });
       const req = httpMock.expectOne('https://api.hsforms.com/submissions/v3/integration/submit/undefined/undefined');
 
       req.flush('');
     })));
-  });
+    it('should include a the hubspot UTK in the request, if it hubspot cookie exists', () => {
+      doc.cookie = 'hubspotutk=mycookie;'
+      const submission = { email: 'email@email.com', firstName: 'John', lastName: 'Doe' };
+      newsletterService.send(submission).subscribe();
+
+      const req = httpMock.expectOne((req: HttpRequest<any>) => {
+        expect(req.url).toEqual('https://api.hsforms.com/submissions/v3/integration/submit/undefined/undefined');
```

Comment:
These expects could be pushed out of the expectOne, let me know if this looks appropriate.